js/script.js

```diff
@@ -0,0 +1,263 @@
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it 1960s.
+            </p><br>
+            <p class="hide">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it 1960s with the 
+              releax map lapora verita.
+            </p>
+            <div class="btns-div">
+            <a href="${card.linkLive}"><img src="./images/See Live.png" class="img-btn" alt="Live Link"></a>
+            <a href="${card.linkSource}"><img src="./images/See Source.png" class="img-btn" alt="source code"></a>
+      
+  
+      </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  
+`;
+  return ('');
+});
+
+// Get the modal
+
+const modal = document.getElementById('myModal');
+const span = document.getElementsByClassName('close')[0];
+span.onclick = () => {
+  modal.style.display = 'none';
+};
+// Get the button that opens the modal
+const proBtn = document.getElementsByClassName('projbtn');
+
+Array.from(proBtn).forEach((button) => {
+  button.addEventListener('click', () => {
+    // Perform desired action here
+    document.body.classList.toggle('no-scroll');
+    modal.style.display = 'block';
+  });
+});
+const modalClose = document.querySelector('.close');
+modalClose.addEventListener('click', () => {
+  modal.classList.remove('act');
+  document.body.classList.remove('no-scroll');
+});
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = (event) => {
+  if (event.target === modal) {
+    modal.style.display = 'none';
+  }
+};
+
+// const allProjects = [
+//   {
+//     id: 'myworks',
+//     name: 'My Recent Works',
+//     description: 'A daily selection of privately
+//     personalized reads; no accounts or sign-ups required.',
+//     featuredImage: '.images/myworks-desktop.png',
+//     technologies: ['HTML', 'BootStrap', 'Ruby'],
+//     linkLive: '<a href="https://Mike111222.github.io/MyPortfolio/" target="blank">See Live</a>',
+//     linkSource: '<a href="https://github.com/Mike111222/MyPortfolio/" target="blank">See Source</a>',
+//   },
+//   {
+//     id: 'proj1',
+//     name: 'Professional Art Printing Data',
+//     description: 'A daily selection of privately personalized reads;
+//      no accounts or sign-ups required. Has been the industry standards.',
+//     featuredImage: './images/Img Placeholder-2',
+//     technologies: ['HTML', 'CSS', 'BootStrap'],
+//     seeProject: 'See Project',
+//     linkLive: '<a href="https://Mike111222.github.io/MyPortfolio/" target="blank">See Live</a>',
+//     linkSource: '<a href="https://github.com/Mike111222/MyPortfolio/" target="blank">See Source</a>',
+//   },
+//   {
+//     id: 'proj2',
```

Comment:
- [ ] Please remove all commented(unused) code from your codebase because it's a good practice to make your code more readable and clean from any unused code.